--- libmng/makefiles/makefile.linux.orig	Sat Jul  1 15:10:35 2000
+++ libmng/makefiles/makefile.linux	Sat Jul  1 15:14:52 2000
@@ -13,19 +13,19 @@
 OPTIONS = -DMNG_BUILD_SO
 
 # where "make install" puts libmng.a,libmng.so*,libmng.h,libmng_conf.h,libmng_types.h
-prefix=/usr/local
+prefix=/usr
 
 # Where the zlib library and include files are located
-ZLIBLIB=../zlib
-ZLIBINC=../zlib
+ZLIBLIB=/usr/lib
+ZLIBINC=/usr/include
 
 # Where the jpeg library and include files are located
-JPEGLIB=../jpgsrc
-JPEGINC=../jpgsrc
+JPEGLIB=/usr/lib
+JPEGINC=/usr/include
 
 # Where the lcms library and include files are located
-LCMSLIB=../lcms/lib
-LCMSINC=../lcms/source
+LCMSLIB=/usr/lib
+LCMSINC=/usr/include
 
 ALIGN=
 # for i386:
@@ -37,7 +37,7 @@
 
 # for pgcc version 2.95.1, -O3 is buggy; don't use it.
 
-CFLAGS=-I$(ZLIBINC) -I$(JPEGINC) -I$(LCMSINC) -Wall -O3 -funroll-loops \
+CFLAGS=-I$(ZLIBINC) -I$(JPEGINC) -I$(LCMSINC) -Wall $(RPM_OPT_FLAGS) \
 	$(OPTIONS) $(ALIGN) # $(WARNMORE) -g
 LDFLAGS=-L. -Wl,-rpath,. \
 	-L$(ZLIBLIB) -Wl,-rpath,$(ZLIBLIB) \
